src/main/java/com/denizenscript/depenizen/bukkit/commands/mythicmobs/MythicSpawnCommand.java

```diff
@@ -1,37 +1,44 @@
 package com.denizenscript.depenizen.bukkit.commands.mythicmobs;
 
-import com.denizenscript.denizencore.objects.Argument;
-import com.denizenscript.depenizen.bukkit.bridges.MythicMobsBridge;
 import com.denizenscript.denizen.objects.LocationTag;
-import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
 import com.denizenscript.denizencore.objects.core.ElementTag;
 import com.denizenscript.denizencore.scripts.ScriptEntry;
 import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.scripts.commands.generator.ArgDefaultText;
+import com.denizenscript.denizencore.scripts.commands.generator.ArgLinear;
+import com.denizenscript.denizencore.scripts.commands.generator.ArgName;
+import com.denizenscript.denizencore.scripts.commands.generator.ArgPrefixed;
 import com.denizenscript.denizencore.utilities.debugging.Debug;
+import com.denizenscript.depenizen.bukkit.bridges.MythicMobsBridge;
 import com.denizenscript.depenizen.bukkit.objects.mythicmobs.MythicMobsMobTag;
 import io.lumine.mythic.api.mobs.MythicMob;
+import io.lumine.mythic.api.mobs.entities.SpawnReason;
 import io.lumine.mythic.bukkit.BukkitAdapter;
 import org.bukkit.entity.Entity;
 
+import java.util.Set;
+
 public class MythicSpawnCommand extends AbstractCommand {
 
     public MythicSpawnCommand() {
         setName("mythicspawn");
-        setSyntax("mythicspawn [<name>] [<location>] (level:<#>)");
-        setRequiredArguments(2, 3);
+        setSyntax("mythicspawn [<name>] [<location>] (level:<#>) (reason:<#>)");
+        setRequiredArguments(2, 4);
+        autoCompile();
     }
 
     // <--[command]
     // @Name MythicSpawn
-    // @Syntax mythicspawn [<name>] [<location>] (level:<#>)
+    // @Syntax mythicspawn [<name>] [<location>] (level:<#>) (reason:<#>)
     // @Group Depenizen
     // @Plugin Depenizen, MythicMobs
     // @Required 2
-    // @Maximum 3
+    // @Maximum 4
     // @Short Spawns a MythicMob at a location.
     //
     // @Description
     // This allows you to spawn a MythicMob at a location using the mob's internal name.
+    // (Requires paper) Additionaly, you may specify a spawn reason for this MythicMob, which can be any of these: <@link url https://hub.spigotmc.org/javadocs/spigot/org/bukkit/event/entity/CreatureSpawnEvent.SpawnReason.html>
```

Comment:
`Paper`, capital p
`additionally` has two `l`s

Reply:
Also, specify the `(requires Paper)` at the end or middle of the line, rather than at the start (to avoid the implication that the command itself requires paper)